workflow/scripts/sequence_logos.py

```diff
@@ -0,0 +1,97 @@
+###############################################################################
+#
+#   Script to plot sequence logos for motifss
+#
+#   AUTHOR: Krish Agarwal
+#   AFFILIATION: University_of_Basel
+#   CONTACT: akrish136@gmail.com
+#   CREATED: 02-08-2020
+#   LICENSE: Apache_2.0
+#
+###############################################################################
+
+import pandas as pd
+import matplotlib.pyplot as plt
+import sys
+import os
+import logomaker
+from argparse import ArgumentParser, RawTextHelpFormatter
+
+##### Using argparse to get input from command line #####
+parser = ArgumentParser(description=__doc__, formatter_class=RawTextHelpFormatter)
+
+parser.add_argument(
+    "--input_files",
+    nargs="+",
+    dest="input_files",
+    help="input files for sequence logos",
+    required=True,
+    metavar="FILE",
+)
+
+parser.add_argument(
+    "--output_location",
+    dest="output_location",
+    help="location where the png logos will be saved",
+    required=True,
+    metavar="DIR",
+)
+
+try:
+    options = parser.parse_args()
+except Exception:
+    parser.print_help()
+
+if len(sys.argv) == 1:
+    parser.print_help()
+    sys.exit(1)
+
+#### Storing commnad line arguments in variables ####
+input_files = options.input_files
+output_location = options.output_location
+
+sorted(input_files)
```

Comment:
`sorted` in Python returns a new sorted object, not sorts in place, so you have to do: `input_files = sorted(input_files)`

Reply:
Right!